ios/.gitignore

```diff
@@ -0,0 +1,97 @@
+test-report.*
```

Comment:
This is the gitignore we use on iOS SDK. Originally came from [here](https://github.com/github/gitignore/blob/main/Swift.gitignore)

Reply:
👍